phonon/update.py

```diff
@@ -320,7 +308,7 @@ def _merge(self):
         update_doc = UpdateDoc(self.__get_cached_doc())
         self.merge(update_doc)
 
-    def _cache(self, block=None):
```

Comment:
Removed blocking in the interface to force people not to expect to be allowed to block. This does funny things to the ioloop.